data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/opponent_weapon_has_no_special.cfg

```diff
@@ -89,3 +89,89 @@
         {SUCCEED}
     [/event]
 )}
+
+#####
```

Comment:
Pentarctagon is trying to move to having separate files for each separate test. I haven't mentioned that when a PR touches a file that already has multiple tests in, but in this case it's a file with a single test.

Reply:
moved to separate file